packages/plugin-core/src/components/lookup/LookupProviderV3Interface.ts

```diff
@@ -53,7 +53,15 @@ export type OnUpdatePickerItemsOpts = {
 };
 
 export type ILookupProviderOptsV3 = {
+  /**
+   * should provide `Create New`
+   */
   allowNewNote: boolean;
+  /**
+   * should provide `Create New with Template`
+   * `allowNewNote` should be true for this to take effect.
```

Comment:
can we add in the comments what the default value is since it's an optional parameter? 

also nit: instead of 'should be true', we should say 'must be true'.